src/main/thrift/parquet.thrift

```diff
@@ -982,6 +1054,23 @@ union ColumnOrder {
    *       `-0.0` should be written into the min statistics field.
    */
   1: TypeDefinedOrder TYPE_ORDER;
+
+  /**
+   * The order only applies to GEOMETRY logical type.
+   *
+   * Please note that geometry objects cannot be compared directly. This order aims to
+   * provide an approach to build a bounding box for geometry objects in the same page
+   * or column chunk.
+   *
+   * In this order, all 2D geometries are regarded as a collection of coordinate (x, y).
+   * For example, POINT has one coordinate, LINESTRING has two coordinates, and POLYGON
+   * might have three or more coordinates. A bounding box is the combination of x_min,
+   * x_max, y_min, and y_max of all coordinates from all geometry values. For simplexty,
+   * min_value field in the Statistics/ColumnIndex is encoded as the concatenation of
+   * PLAIN-encoded DOUBLE-typed x_min and y_min values. Similarly, max_value field is
+   * encoded as the concatenation of PLAIN-encoded DOUBLE-typed x_max and y_max values.
```

Comment:
Are there possibly other encodings that might be more useful here (I seem to recall that using interleaved encoding (i.e. z-order) might be useful for searching but I'd have to go back and do some reading to confirm.

Reply:
CMIW, z-order or hilbert curve apply to points only. We might need to obtain a point (e.g. centroid) from geometries to apply space-filling curve but I suspect the usability compared to bounding box.

Reply:
+1. z-order or hilbert curve apply to points only. Unless we use something like XZ2

Reply:
Here the column statistics are the aggregate index for a group of geometry values. For example, usually the column statistics are collected from a data page of 10,000 values where bbox is more suitable. CMIW, S2/H3 covering might be more efficient in spatial join or record-level filtering.